talisker/gunicorn.py

```diff
@@ -47,6 +49,17 @@
 }
 
 
+logger = logging.getLogger(__name__)
```

Comment:
This probably needs to be moved back to where it was, or else it will be created at import time, which may be *before* the logging has been configured, meaning the logger will be the wrong type, and it won't support tags on logs.

Note: this is only an issue for talisker's own logs, as talisker shouldn't import any application code until after it has configured logging. But in talisker's own code, we need to delay log creation, as some things will be imported before logging has been configured.